addons/binding/org.openhab.binding.netatmo/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="netatmo"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+                  
+    <channel-type id="lastStatusStore">
```

Comment:
with this amount of channels, I would think that you should mark quite a lot of them as "advanced", so that the UI has a chance to show the most important stuff first.